varnishgather

```diff
@@ -428,6 +440,46 @@ done
 
 shift $((OPTIND - 1))
 
+if [ -z "$NAME" ]; then
+    VARNISHD_CMD=$(ps -o cmd= -p $(pgrep varnishd))
```

Comment:
this would generate multiple values when multiple `varnishd` processes are running, wouldn't it?

also, you wrote `get_varnishd_instances`, it feels like a shame to not use it?

Reply:
I changed this entire block to tighten up sloppy work on my part.